### PR TITLE
Myo test

### PR DESCRIPTION
**Pull request by m-aung**
This pull request includes:
- All together 21 test added to the main_test.ts 
- Join.test file has new test cases.
- testing.md is edited.
- [x] current version passed all 21 test cases.
21 test passed after adding joins method

### CODE REVIEW

File: lib/draft.ts [hunk 1]
```diff
@@ -304,22 +280,36 @@ export class Dorm {
...
-    joinList.forEach(el => {
-      if(el.table === target) el.table = arg;
-      if(el.on && el.on.includes(target)) {
-        el.on.split(' ').forEach((word:string) =>  {
-          if(word === target) word = arg;
-        })
-      }
-    })
-    }
+  // as(arg:string, target:string){
```
Comment (chaekmh) [inline lib/draft.ts]: Hey myo, 

Make sure to work on As method in your different feature branch and only PR working code.  We don't want to put notworking/unrelated work on the production level codebase.

File: lib/draft.ts [hunk 2]
```diff
@@ -55,7 +55,7 @@ export class Dorm {
       action: {
         type: null,
         table: null,
-        columns: '*',
+        columns: '*', // {origin: height, as: h}
```
Comment (chaekmh) [inline lib/draft.ts:58]: Myo, Can you edit this please?

**Review by chaekmh — CHANGES_REQUESTED** [review]
Just one small thing

File: lib/draft.ts (hunk 2 above)
Comment (m-aung) [reply lib/draft.ts:58]: sure.

**Review by chaekmh — APPROVED** [review]

**Pull request merged**